src/components/calcite-shell-center-row/calcite-shell-center-row.scss

```diff
@@ -10,6 +10,8 @@
   --calcite-app-shell-center-row-height-small: 33%;
   --calcite-app-shell-center-row-height-medium: 70%;
   --calcite-app-shell-center-row-height-large: 100%;
+
+  --calcite-app-shell-center-row-border: 1px solid var(--calcite-app-border);
```

Comment:
Sorry, I meant to suggest a sass var for this.

Reply:
Oh. Is it better as a sass var?

Reply:
@jcfranco I'm down to do that but we would wanna update all the sass files to use that var, yeah?